src/lib/support/HexUtils.h

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Please try very hard to avoid 'Util(s)' or 'Helper(s)' in the file names. More explicit naming is easier to work with. 

I suggest 'HexParsing' or just 'Hex'